test/unittests/mongoBackend/mongoContextProvidersQuery_test.cpp

```diff
@@ -1451,27 +2065,27 @@ TEST(mongoContextProvidersQueryRequest, severalCprs1)
   EXPECT_EQ("", RES_CER_ATTR(0, 0)->providingApplication);
 
   EXPECT_EQ("A2", RES_CER_ATTR(0, 1)->name);
-  EXPECT_EQ("T", RES_CER_ATTR(0, 1)->type);
+  EXPECT_EQ("", RES_CER_ATTR(0, 1)->type);
```

Comment:
Important to know, this has changed regarding last time you check these tests.

Attribute type is not returned in the case of forwarded attributes but take into account you don't need it. I mean, when you forward a queryContext to a potential CPr, you cannot include the attribute type in the queryContext.